rename.cc

```diff
@@ -0,0 +1,54 @@
+#include <fcntl.h>
+#include <sys/stat.h>
+
+#include "./capsicum-test.h"
+#define RENAME_TEST_FILENAME        ".rename_test"
+#define RENAMEAT_TEST_FILENAME      ".renameat_test"
+#define RENAMEAT_TEST_DIR           ".renameat_testdir"
+
+/*
+added to test the renameat syscall for the case that
+    - the "to" file already exists
+    - the "to" file is specified by an absolute path
+    - the "to" file descriptor is used
```

Comment:
If the "to" path is absolute, then isn't the "to" file descriptor effectively ignored?